config/locales/views/reactions/en.yml

```diff
@@ -10,7 +10,7 @@ en:
         thumbsup: Thumbs up
         thumbsdown: Thumbs down
         unicorn: Unicorn
-        vomit: Vomit
+        vomit: Flag
```

Comment:
Do you think we should rename this actual key to "flag" instead of "vomit" so that it doesn't cause confusion to anyone in the future? Same applies to fr.yml

Reply:
I've left it the same at the moment, as the category of the reaction under the hood is still 'vomit' 😕  e.g. in some places we were displaying `reaction.category.capitalize`, whereas now we can display `t("views.reactions.category.${reaction.category}")`, and 'vomit' will map to 'Flag'.

I think it may be worth some later refactoring so that we just don't use "vomit" at all, which would help avoid confusion, and allow us to easily change this i18n key too.